lms/static/sass/base/_grid-settings.scss

```diff
@@ -4,10 +4,13 @@
 $max-width: 1200px;
 /* Override the default global box-sizing */
 $border-box-sizing: false;
+/* Grid width variables */
+$large-min-width: 769px;
```

Comment:
I want the "View all courses" link to be right aligned for min-width when the `$large` breakpoint begins. I don't want that behaviour to be limited by the upper limit of the `$large` breakpoint.
If I do what you've suggested, the link would be aligned in the middle of the parent container for the `huge` breakpoint.

This way, I've only set the `min-width` for the media query in question. To make in more concise, I've used the variable for both media query and the `$large` breakpoint, just in case if we ever changed the lower limit of the `$large` breakpoint.

As always, I'm open to any suggestions on the matter.